doc/newfeat.texi

```diff
@@ -59,6 +59,14 @@ The following have been made obsolete, see their documentation for more detail:
 
 @itemize @bullet
 
+@item Libraries for literate data analysis are obsolete and not loaded by default.
+This includes @code{ess-noweb}, @code{ess-swv}, and related
+functionality like @code{Rnw-mode}. Users are encouraged to switch to
+one of several other packages that deal with these modes. For example,
+polymode @url{https://github.com/polymode/poly-R/},
+@url{https://polymode.github.io/}, or markdown-mode with edit-indirect
+@url{https://jblevins.org/projects/markdown-mode}.
```

Comment:
Markdown mode (without polymode etc) really is not a substitute for ess-noweb:
1. It does not deal with *.Rnw at all.
2. The most important functionality for *.Rmd and *.Rnw is a good user interface to send R code to the R process, current chunk, all chunks,  all chunks above 'point', etc,  but also to Sweave, knit, tangle, or purl the whole file.. (and of course, moving from chunk to chunk etc but that's almost too fundamental to be noteworthy) *and* `ess-noweb` has most of that in its mode specific menu  `Noweb` including submenus.

So I don't see "several" packages.  It's the part of the polymode bundle you get via poly-R  only, AFAICS.

Reply:
Markdown-mode lets you edit code blocks similar to how org-mode does: you do `C-c '` which pops up a new buffer in the proper major mode. Since many Emacs users also use org-mode, this is pretty intuitive.

Reply:
Yes (about editing), and indeed thank you for teaching me about `C-c '`.  But that's only one important feature when working with *.Rmd.  The other is interaction with the *R* process buffer. Sending chunks (one by one, conveniently /  all chunks above point / all chunks / knit the whole document / render the whole document / .. etc.    Things that were even missing with polymode-R (forgot the correct name) when I last looked (which was several months ago). *And* for people not using this daily, having nice Emacs menus, which show you possibilities and simultaneously mention the key strokes.

Reply:
All the things that you mention (menus, evaluation, navigation) are now in polymode since end of September after the rewrite. Might not yet be perfect, but things could always be smoothed by next release of ESS.